src/test/scala/spark/dicom/deidentifier/TestDicomDeidentifier.scala

```diff
@@ -0,0 +1,114 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package ai.kaiko.spark.dicom.deidentifier
+
+import ai.kaiko.spark.dicom.deidentifier.DicomDeidentifier._
+import ai.kaiko.dicom.DicomDeidentifyDictionary.{
+  DUMMY_DATE,
+  DUMMY_TIME,
+  DUMMY_DATE_TIME,
+  EMPTY_STRING,
+  DUMMY_STRING
+}
+import ai.kaiko.spark.WithSpark
+import org.apache.log4j.Level
+import org.apache.log4j.LogManager
+import org.dcm4che3.data.Keyword.{valueOf => keywordOf}
+import org.dcm4che3.data._
+import org.scalatest.CancelAfterFailure
+import org.scalatest.DoNotDiscover
+import org.scalatest.funspec.AnyFunSpec
+import java.io.File
+
+
+object TestDicomDeidentifier {
+  val SOME_DICOM_FILEPATH =
+    "src/test/resources/Pancreatic-CT-CBCT-SEG/Pancreas-CT-CB_001/07-06-2012-NA-PANCREAS-59677/201.000000-PANCREAS DI iDose 3-97846/1-001.dcm"
+  lazy val SOME_DICOM_FILE = {
+    val file = new File(SOME_DICOM_FILEPATH)
+    assert(file.exists)
+    file
+  }
+}
+
+@DoNotDiscover
```

Comment:
Feels a bit odd to me not being able to run this test class by itself.

Reply:
Maybe there is a better way of doing it, but without the decorator, the test would run twice. Once on it's own, and once as part of the suite